changelog.d/10321.removal

```diff
@@ -0,0 +1 @@
+Remove undocumented `room_alias` key from `/createRoom` response. Contributed by @thezaidbintariq.
```

Comment:
The number in the changelog's filename will need to match the number of your PR (14884). Could you rename the changelog file to match that number?

Reply:
@anoadragon453 Done, could you please verify if I did it correctly?